.github/workflows/image-scan.yml

```diff
@@ -0,0 +1,202 @@
+        if-no-files-found: error
+
+  # Evaluate scan results.
+  evaluate-results:
+    needs: [ lacework-scan ]
+    runs-on: ubuntu-latest
+    steps:
+    - name: Download all results from artifacts
+      uses: actions/download-artifact@v3
+
+    - name: Build and set job summary
+      shell: bash
+      run: |
+        lwResultFiles="$(find . -type f -name '*.lacework-result.json' | sort)"
+
+        echo "## <a name=\"scan-results\"></a>Lacework scan results" >> $GITHUB_STEP_SUMMARY
+        for jsonFile in $lwResultFiles; do
+          image=$(jq -r '.image' $jsonFile)
+          cveC=$(jq -r '.stats.Critical.total' $jsonFile)
+          cveH=$(jq -r '.stats.High.total' $jsonFile)
+          cveM=$(jq -r '.stats.Medium.total' $jsonFile)
+          cveL=$(jq -r '.stats.Low.total' $jsonFile)
+          cveI=$(jq -r '.stats.Info.total' $jsonFile)
+        
+          echo "" >> $GITHUB_STEP_SUMMARY
+          echo "### $image" >> $GITHUB_STEP_SUMMARY
+
+          # Highlight number in table cell.
+          _hln() {
+            if [ ${1} -gt 0 ]; then
+              echo "**${1}** :x:"
+            else
+              echo "${1} :heavy_check_mark:"
+            fi
+          }
+
+          echo "| Critical | High | Medium | Low | Info |" >> $GITHUB_STEP_SUMMARY
+          echo "|---|---|---|---|---|" >> $GITHUB_STEP_SUMMARY
+          echo "| $(_hln $cveC) | $(_hln $cveH) | $cveM | $cveL | $cveI |" >> $GITHUB_STEP_SUMMARY
+          echo "" >> $GITHUB_STEP_SUMMARY
+
+          # Print CVE table (if not empty).
+          cveCount=$(jq -r '.cves | length' $jsonFile)
+          if [ $cveCount -gt 0 ]; then
+            echo "|CVE|Severity|Package|Current version|Fix version| Status|" >> $GITHUB_STEP_SUMMARY
+            echo "|---|---|---|---|---|---|" >> $GITHUB_STEP_SUMMARY
+            for row in $(jq -r '.cves[] | @base64' $jsonFile); do
+              row=$(echo ${row} | base64 --decode)
+              _jq() {
+                echo $row | jq -r ${1}
+              }
+              echo "| [$(_jq '.cveId')]($(_jq '.link')) | $(_jq '.severity') | $(_jq '.package') | $(_jq '.currentVersion')|  | $(_jq '.status') |" >> $GITHUB_STEP_SUMMARY
+            done
+          fi
+        done
+
+    - name: Build PR comment
+      if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
+      id: build-pr-comment
+      shell: bash
+      run: |
+        lwResultFiles="$(find . -type f -name '*.lacework-result.json' | sort)"
+        
+        # Collect overall status.
+        lwStatusIcon=":green_circle:"
+        for jsonFile in $lwResultFiles; do
+          cveC=$(jq -r '.stats.Critical.total' $jsonFile)
+          cveH=$(jq -r '.stats.High.total' $jsonFile)
+          if [ $cveC -gt 0 ] || [ $cveH -gt 0 ]; then
+            lwStatusIcon=":red_circle:"
+            break
+          fi
+        done
+
+        c="### ${lwStatusIcon}&nbsp; Lacework scan results:"
```

Comment:
`c` collects the final result? I'd maybe expand the name to something more descriptive like `result` since it keeps being used a lot of times.